projects/ama-testing/src/e2e/pages/process-properties.card.ts

```diff
@@ -79,6 +79,13 @@ export class ProcessPropertiesCard extends GenericPage {
         await super.click(connectorActionOption);
     }
 
+    async checkConnectorInList(connectorName: string) {
+        await super.click(this.connectorSelector);
+        const connectorOption = element(by.cssContainingText('.mat-option-text', connectorName));
+        const isConnector = connectorOption ?  true :  false;
```

Comment:
`return !!connectorOption;`

Reply:
Could we use:
return await super.waitForElementToBeVisible(connectorOption);
?